analyzers/src/SonarAnalyzer.CSharp/Rules/ObjectCreatedDropped.cs

```diff
@@ -25,31 +25,27 @@
 using Microsoft.CodeAnalysis.Diagnostics;
 using SonarAnalyzer.Helpers;
 
-namespace SonarAnalyzer.Rules.CSharp
+namespace SonarAnalyzer.Rules.CSharp;
+
+[DiagnosticAnalyzer(LanguageNames.CSharp)]
+public sealed class ObjectCreatedDropped : SonarDiagnosticAnalyzer
 {
-    [DiagnosticAnalyzer(LanguageNames.CSharp)]
-    public sealed class ObjectCreatedDropped : SonarDiagnosticAnalyzer
-    {
-        internal const string DiagnosticId = "S1848";
-        private const string MessageFormat = "Either remove this useless object instantiation of class '{0}' or use it.";
+    private const string DiagnosticId = "S1848";
+    private const string MessageFormat = "Either remove this useless object instantiation of class '{0}' or use it.";
 
-        private static readonly DiagnosticDescriptor rule =
-            DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
+    private static readonly DiagnosticDescriptor Rule = DescriptorFactory.Create(DiagnosticId, MessageFormat);
```

Comment:
Just FYI: 
We'll do a batch search/replace on these to use the DescriptorFactory everywhere soon. It might lead to a rebase conflict.